src/css/screens/videocard.css

```diff
@@ -61,4 +61,18 @@
 }
 .filled-like{
     color: var(--red-color);
+}
+.videocard-play{
+    position: absolute;
+    right: 140px;
+    top: 50px;
+    cursor: pointer;
+    width: 40px !important;
+    height: 40px !important;
```

Comment:
important in css is discouraged, do try to avoid using it in direct styles. Why is it used here by the way?

Reply:
to overwrite styles of material icons, one has to use !important.